types/types.go

```diff
@@ -617,10 +617,11 @@ type PlacementPreferences struct {
 
 // ServiceNetworkConfig is the network configuration for a service
 type ServiceNetworkConfig struct {
-	Priority    int      `yaml:",omitempty" json:"priotirt,omitempty"`
-	Aliases     []string `yaml:",omitempty" json:"aliases,omitempty"`
-	Ipv4Address string   `mapstructure:"ipv4_address" yaml:"ipv4_address,omitempty" json:"ipv4_address,omitempty"`
-	Ipv6Address string   `mapstructure:"ipv6_address" yaml:"ipv6_address,omitempty" json:"ipv6_address,omitempty"`
+	Priority     int      `yaml:",omitempty" json:"priotirt,omitempty"`
```

Comment:
Is `priotirt` a typo? I did not change this in my PR but it shows a diff due to reformatting. Happy to fix this as well.

Reply:
definitely a typo and yes you can add the fix in your PR, thanks

Reply:
Fixed, thanks for confirming!